src/Avalonia.Base/Platform/IDrawingContextImpl.cs

```diff
@@ -177,6 +166,10 @@ public interface IDrawingContextImpl : IDisposable
         /// Attempts to get an optional feature from the drawing context implementation
         /// </summary>
         object? GetFeature(Type t);
+
+        void PushRenderOptions(RenderOptions renderOptions);
```

Comment:
Why do we need Push/Pop if there is a direct setter?

Reply:
Y this can be removed and we can just use the local value. I am still not sure if we need value inheritance here. Previously BitmapInterpolationMode was an attached property this isn't ideal if we need to read the value on every visual.